manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/notebooks/tree/main/codeserver"
-    opendatahub.io/notebook-image-name: "Code Server"
-    opendatahub.io/notebook-image-desc: "Code Server workbench image, allows to run Visual Studio Code (VSCode) on a remote server through the browser."
+    opendatahub.io/notebook-image-name: "code-server"
+    opendatahub.io/notebook-image-desc: "code-server workbench image, allows to run code-server on a remote server through Web."
```

Comment:
@jstourac @grainnejenningsRH I don't much like this formulation. Sounds like I need some additional server.

The text shows in the web ui here

![image](https://github.com/opendatahub-io/notebooks/assets/442720/320cd8c9-fa3c-42c1-8721-0af82c729a25)

Reply:
I agree, probably something like: _Code server workbench allows users to code, build, and collaborate on projects directly from a server._